examples/example.js

```diff
@@ -240,8 +244,16 @@ const main = async (cmd) => {
   switch (cmd) {
     case "create-wallet":
       salt = process.argv[3] || 0;
-      let smartWalletAddress = await createWallet(owner, token, process.argv[3], process.argv[4]);
-      console.log("generated smart wallet", smartWalletAddress)
+      let smartWalletAddress = await createWallet(
+        owner,
+        token,
+        process.argv[3],
+        process.argv[4]
+      );
+      console.log(
+        `A new smart wallet with salt ${salt} is created for ${owner}:\nResponse:\n`,
+        smartWalletAddress
+      );
```

Comment:
In this `create-wallet` command, what should the API return if we call create-wallet with `salt: '1'` twice? In my opinion, we should call this `get-wallet`, instead of `create-wallet`, because a typical create command would fail, if the wallet already exists, but a get command should return the wallet details.

Also, in reality we are just returning them a smart wallet address with a salt. 

And I would keep the return exactly the same as the get wallet addresses command, with `{address, salt, factory}`, to give them the full details.

Current output is:
 { address: '0x40Baaa1657e7D4cF5d949FCAc0a88cBB4660050e' }

Reply:
okay, we can keep it `create-wallet` for now.